core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml

```diff
@@ -1,8 +1,8 @@
 templates:
   main: !template
-    reportTemplate: 'dummy.jxml'
+    reportTemplate: 'dummy.jrxml'
     attributes:
-      mapDef: !map
+      map: !map
```

Comment:
Was mapDef used to test the none default map attribute ?

Reply:
I don't understand the question!

Reply:
The `inputMapper`/`outputMapper` are well tested in the `multiple_maps` and `oereb` example, no need to be tested here.
We will keep the example as simple as possible.